sidecar/config/config.go

```diff
@@ -219,14 +219,26 @@ func (c *Config) loadFromContext(context *cli.Context) error {
 				hcType = HTTPSHealthCheck
 			case "tcp":
 				hcType = TCPHealthCheck
+			case "cmd":
```

Comment:
I'm wondering if the standardized [file URI scheme](https://en.wikipedia.org/wiki/File_URI_scheme) (e.g., `file:///usr/bin/health.sh`) could be a better fit here.